src/iterative_sorting/iterative_sorting.py

```diff
@@ -6,20 +6,36 @@ def selection_sort( arr ):
         smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
-             
-
-
-
+        for j in range(cur_index, len(arr)):
```

Comment:
very clean implementation; going through the array as you look for the smallest value